doc/run_experiment.rst

```diff
@@ -506,8 +506,18 @@ fixed_parameters *(Optional)*
 """""""""""""""""""""""""""""
 
 List of dicts containing parameters you want to have fixed for each
-classifier in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used).
+learner in :ref:`learners` list. Any empty ones will be ignored
+(and the defaults will be used). If there is a conflict between parameters
+specified in :ref:`param_grids` and :ref:`fixed_parameters`, an exception
+will be raised (unless the conflicting values are the same). Furthermore,
```

Comment:
I agree, "conflict" might not be the right word here. I just meant that the same parameter is specified in both the fixed parameters and in the parameter grids. If it's just a single value inside a list in the parameter grid and the value is the same as the fixed parameter, it doesn't matter, right? It seems to me that that is basically another way to fix a parameter's value. Anyway, the main point is that I wanted to recognize that situation and not raise an exception there. I can think about rephrasing it.